app/controllers/exchanges/hbx_profiles_controller.rb

```diff
@@ -723,6 +738,38 @@ def aptc_csr_family_index
     end
   end
 
+  def year_plan_data(year)
+    product_query = BenefitMarkets::Products::Product.where(
+      :"application_period.min".lte => Date.new(year, 12, 31),
+      :"application_period.max".gte => Date.new(year, 1, 1)
+    )
+    {
+      year: year,
+      plans_number: product_query.count,
+      pvp_numbers: BenefitMarkets::Products::PremiumValueProduct.where(active_year: year).count,
+      enrollments_number: Family.actual_enrollments_number(year: year),
+      products: product_query.distinct(:kind).map { |kind| kind.to_s.capitalize }.join(", ")
+    }
+  end
+
+  def carrier_data(carrier_data, year)
```

Comment:
I'd consider making this private, too (easily tell actions from utility methods, extent we need to grep for future refactoring, etc.)